### ISSUE TITLE
FNX-4814 ⁃ Improve accessibility of quick action sheet

### ISSUE THREAD

**Issue by sblatz** [mozilla-mobile/fenix#1127]

**Comment by sv-ohorvath** [comment]
@sblatz I tested the quick action bar accessibility with Talkback. I find it weird that the quick action sheet handle is read but it doesn't have the function to collapse/expand the bar. I think it would be expected.
